src/Dobble/DeckValidator.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Marmelab\Dobble;
+
+abstract class DeckValidator
```

Comment:
Looks weird to define such a class as `abstract` as we are going to use its own methods.